next/next.config.js

```diff
@@ -62,6 +62,19 @@ const nextConfig = {
   },
   async redirects() {
     return [
+      {
+        source: '/o-nas/dokumenty-a-zverejnovanie-informacii/:categorySlug/:slug',
+        destination: '/o-nas/dokumenty-a-zverejnovanie-informacii/:slug',
+        permanent: false,
+        locale: false,
+      },
+      {
+        source: '/en/o-nas/dokumenty-a-zverejnovanie-informacii/:categorySlug/:slug',
+        destination: '/en/about-us/documents-and-public-disclosure-of-information/:slug',
+        permanent: false,
+        locale: false,
+      },
+
```

Comment:
I think you can mark both as permanent

```suggestion
      {
        source: '/o-nas/dokumenty-a-zverejnovanie-informacii/:categorySlug/:slug',
        destination: '/o-nas/dokumenty-a-zverejnovanie-informacii/:slug',
        permanent: true,
        locale: false,
      },
      {
        source: '/en/o-nas/dokumenty-a-zverejnovanie-informacii/:categorySlug/:slug',
        destination: '/en/about-us/documents-and-public-disclosure-of-information/:slug',
        permanent: true,
        locale: false,
      },

```

Reply:
Okay